fdb-record-layer-core/src/main/java/com/apple/foundationdb/record/query/plan/plans/RecordQueryTypeFilterPlan.java

```diff
@@ -113,29 +113,30 @@ public String toString() {
         return getInner() + " | " + recordTypes;
     }
 
+    @Nonnull
     @Override
     @API(API.Status.EXPERIMENTAL)
-    public boolean equalsWithoutChildren(@Nonnull RelationalExpression otherExpression) {
-        return otherExpression instanceof RecordQueryTypeFilterPlan &&
-               recordTypes.equals(((RecordQueryTypeFilterPlan)otherExpression).recordTypes);
+    public RecordQueryTypeFilterPlan rebaseWithRebasedQuantifiers(@Nonnull final AliasMap translationMap,
```

Comment:
Where did equalsWithoutChildren go?

Reply:
Alright this is done in the super class now as the record types are exposed through an abstract getter.